### ISSUE TITLE
Options don't work correctly over GET for (at least) ListContributionEventsOptions

### ISSUE THREAD

**Issue by csoriano1618** [xanzy/go-gitlab#555]
Following https://github.com/xanzy/go-gitlab/issues/552 and with the same code example, the resulted url is:

```
/api/v4/events?Action=&Before=&Sort=&TargetType=&page=1&per_page=100
```

Note that all options except "page" and "per_page" are capitalized and they are not omitted if empty. They should both be omitted if empty and being lower case. The difference between "page" and "per_page" and the others is that they provide "url:" field for their default values, whether the other options only provide json. 

In my understanding, when using the query.Values and query.encode in go-querystring with GET they are using the url field, and not the json field, so it needs to provide the url field too for it to work correctly.

The following definition of ListContributionEventsOptions makes it work:
```go
type ListContributionEventsOptions struct {
	ListOptions
	Action     *EventTypeValue       `url:"action,omitempty" json:"action,omitempty"`
	TargetType *EventTargetTypeValue `url:"target_type,omitempty" json:"target_type,omitempty"`
	Before     *string               `url:"before,omitempty" json:"before,omitempty"`
	After      *string               `url:"after,omitempty" json:"after,omitempty"`
	Sort       *string               `url:"sort,omitempty" json:"sort,omitempty"`
}
```

**Comment by svanharmelen** [comment]
Thanks for the report! This is already being addressed in #554 and will most likely be merged tomorrow.

**Comment by csoriano1618** [comment]
thanks!
